### PR TITLE
Updating the license from Apache 2.0 to Business Source License 1.1

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: kitchen.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Mondoo, Inc.
```
Comment (tas50) [inline kitchen.yml]: Chef uses a `Copyright::` format in headers. Cookstyle will enforce it.

Source: I wrote the cop and did the historical digging into why Adam used this format.

```suggestion
# Copyright:: (c) Mondoo, Inc.
```

Comment (benr) [reply kitchen.yml]: Fix coming...

Comment (chris-rock) [reply kitchen.yml]: pushed the change

Comment (tas50) [reply kitchen.yml]: They all need to get updated

**Review by benr — APPROVED** [review]

**Pull request merged**